react-express-mongodb/frontend/src/App.js

```diff
@@ -1,56 +1,55 @@
-import React from 'react';
-import {request} from './utilities/httpRequestHandler'
-import './App.scss';
+import React from "react";
+import axios from "axios";
+import "./App.scss";
 import AddTodo from "./components/AddTodo";
 import TodoList from "./components/TodoList";
 
-export default class App extends React.Component{
-
-  constructor(props){
+export default class App extends React.Component {
+  constructor(props) {
     super(props);
 
     this.state = {
-      todos: []
-    }
+      todos: [],
+    };
   }
 
   componentDidMount() {
-      request('get','/api')
-          .then((response) => {
-              this.setState({
-                  todos: response.data.data
-              })
-          }).catch((e) => console.log('Error : ', e))
+    axios
+      .get("/api")
+      .then((response) => {
+        this.setState({
+          todos: response.data.data,
+        });
+      })
+      .catch((e) => console.log("Error : ", e));
   }
 
-    _handleAddTodo = (value) => {
-      request('post', '/api/todos', {text:value})
-          .then((response) => {
-              let todosCopy = this.state.todos;
-              todosCopy.unshift({text:value});
```

Comment:
here we are mutating an element of the state (not a copy) which is something we shouldn't do with react